src/main/java/org/kakara/engine/item/features/HorizontalRotation.java

```diff
@@ -0,0 +1,12 @@
+package org.kakara.engine.item.features;
+
+import org.joml.Quaternionf;
+import org.kakara.engine.item.GameItem;
+import org.kakara.engine.renderobjects.RenderChunk;
+
+public class HorizontalRotation implements Feature {
```

Comment:
I am going to ask for the removal of this class due to the fact it is hard coded. This is not very useful the way it is.